internal/npm_install/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//internal:__subpackages__"])
```

Comment:
why does this matter if there are no rules in the package?

Reply:
you mean the package()? yeah. That can go.

Reply:
whole file can just be
`# Marker that this directory is a bazel package`

Reply:
You still need the BAZEL.build file in the folder if there is a .bzl file there right? But it can be an empty file. Is the convention to just leave it empty?